pytest_spark/__init__.py

```diff
@@ -34,7 +34,7 @@ def pytest_configure(config):
         SparkConfigBuilder().initialize(options_from_ini=spark_options)
 
 
-def pytest_report_header(config, startdir):
+def pytest_report_header(config, start_path):
```

Comment:
```suggestion
def pytest_report_header(config, *args):
```

Since this is unused, I think we should be more future proof. I also made a PR at the same time you did, but I'll close it in favor of this one. (see #26)

Reply:
knowing that you could simply remove the parameter, i think it is the best option to make it optional, as this is the most subtle modification to a package that hasn't seen an update for the last 4 years.
introducing *args (and why not also **kwargs) introduces more complications in the long run and should be left as a last resort.
fyi: pytest 8.1.0 was yanked, so this isn't this urgent anymore